benchmark/src/benchmark/measure.clj

```diff
@@ -103,23 +126,95 @@
+    (vec (map (fn [f] (conj {:name (-> f :context :dh-config :name)
+                             :db-entities (:db-entities (:context f))}
+                            (select-keys (:time f) [:mean :median :std])))
+              figs)))
+
+  (def configs (map #(assoc-in % [:config :keep-history?] true) c/db-configs))
+  (def opts {:output-format "edn"
+             :iterations 10,
+             :tag #{test}
+             :db-entity-counts [0 1000 10000 100000 1000000]})
+
+  (defn ret-wanderung-export-db-fn [conn]
+    (ret-export-db migrate/export-db-wanderung conn "/tmp/dh.wanderung.dump"))
+  (def wanderung-figures
+    (get-measurements opts configs {:spec-fn-name "export-db-wanderung"
+                                    :make-fn-invocation ret-wanderung-export-db-fn}))
+  (def wanderung-figs-filtered (filter-export-figures wanderung-figures))
+  (spit "wanderung-figures" wanderung-figures)
+  (spit "wanderung-figs-filtered" wanderung-figs-filtered)
+
+  (def test-opts {:output-format "edn"
+                  :iterations 10,
+                  :tag #{test}
+                  :db-entity-counts [0 10 100]})
+
+  (def test-wanderung-figures
+    (get-measurements test-opts configs {:spec-fn-name "export-db-wanderung"
+                                         :make-fn-invocation ret-wanderung-export-db-fn}))
+  (def test-wanderung-figs-filtered (filter-export-figures test-wanderung-figures))
+  test-wanderung-figures
+  test-wanderung-figs-filtered
+
+  (defn ret-tc-export-db-fn [conn]
+    (ret-export-db migrate/export-db-tc conn "/tmp/dh.tc.dump"))
+  (def tc-figures
+    (get-measurements opts {:spec-fn-name "export-db-tc"
+                            :make-fn-invocation ret-tc-export-db-fn}))
+  (def tc-figs-filtered (filter-export-figures tc-figures))
+  (spit "tc-figures.edn" tc-figures)
+  (spit "tc-figs-filtered.edn" tc-figs-filtered)
+
+  (def test-tc-figures
+    (get-measurements test-opts configs {:spec-fn-name "export-db-tc"
+                                         :make-fn-invocation ret-tc-export-db-fn}))
+  (def test-tc-figs-filtered (filter-export-figures test-tc-figures))
+  test-tc-figs-filtered
+
+  (defn ret-clj-export-db-fn [conn]
+    (ret-export-db migrate/export-db-clj conn "/tmp/dh.clj.dump"))
+  (def clj-figures
+    (get-measurements opts {:spec-fn-name "export-db-clj"
+                              :make-fn-invocation ret-clj-export-db-fn}))
+  (def clj-figs-filtered (filter-export-figures clj-figures))
+  (spit "clj-figures.edn" clj-figures)
+  (spit "clj-figs-filtered.edn" clj-figs-filtered)
+
+  (def test-clj-figures
+    (get-measurements test-opts configs {:spec-fn-name "export-db-clj"
+                                         :make-fn-invocation ret-clj-export-db-fn}))
+  (def test-clj-figs-filtered (filter-export-figures test-clj-figures))
+  test-clj-figs-filtered
+
+  (alter-var-root #'datahike.api/connect
+                  (fn [f] (fn [& args] (println "instrumented") (apply f args))))
+
+  )
```

Comment:
Should be removed from this place before merging. If you want to keep this code to make further development easier, you could move it to a sandbox namespace in the dev directory and if there's something that might be useful in the future, it should be integrated properly into the codebase (if not already done).

Reply:
Yes, I put it there temporarily in case it turned out to be useful. It's just been removed.

Reply:
Oh, I put it there just in case it turned out to be useful. It's just been removed.

Reply:
There is still a comment section in this namespace, which should be removed before merging.